src/environmental/water_quality/surface_water_quality.py

```diff
@@ -285,20 +314,41 @@ def get_req_surface_water_quality_from_db(
 
     Returns
     -------
-    gpd.GeoDataFrame
+    Optional[gpd.GeoDataFrame]
         A GeoDataFrame containing the latest requested surface water quality data from the database for the
-        specified catchment area.
+        specified catchment area, or `None` if no records are found.
```

Comment:
I think it would be more appropriate to return an empty GeoDataFrame,, what do you think?

Reply:
As requested, this function `get_req_surface_water_quality_from_db()` has been removed.